Examples/Scripts/IcaFixProcessingBatch.sh

```diff
@@ -132,14 +132,14 @@ main() {
 		export FSL_FIXDIR=${FixDir}
 	fi
 
-	# set list of fMRI on which to run ICA+FIX
+	# set list of fMRI on which to run ICA+FIX, separate groups with @ (for MR FIX concatenation)
```

Comment:
I'm thinking that users might be a little confused what is meant by "groups" without more context.  Could we add more explanation, and perhaps a (commented out) example of what `fMRINames` would look like in the multiple "groups" condition.

Part of the reason this could be confusing is that usually when the '@' construction is used, it has to be used between _all_ the inputs (so that there are no whitespaces at all).  In fact, because the situation is different here, perhaps we should use '%' rather than '@' to separate the groups -- which would also match nicely to the separator between groups used in the `--multirun-fix-names` argument of `DeDriftAndResamplePipeline.sh`

Reply:
I think making the usages similar is a good idea if they really do parallel eachother.

Reply:
dedrift takes the string as an input argument, if it used spaces it would need to be quoted, and some pipeline authors don't like being space-safe.  Here, it is in a variable that used to be space delimited in previous versions, and assignment to variables is generally already quoted to work correctly with spaces.  As it currently stands, people can copy their settings from a previous version of this example script and it will work, but if we go with @ and % to match dedrift, then previous settings won't work.

Reply:
What is the verdict here?  Do we make it hard to copy paste between dedrift and this example, or do we make it hard for previous single-run fix users to get started with MR FIX?  Or do we require preserved spaces in the arguments to dedrift instead?

Reply:
It should also be possible to support both `a@b%c@d` and `a b%c d` conventions in this script without much trouble.